tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -348,14 +394,20 @@ Replace data files for amendment
     user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}dates.csv
     user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}dates.meta.csv
     user clicks button  Upload data files
-    user checks headed table body row cell contains  Status          2  Complete   wait=180
+    user opens details dropdown  Footnote 2 seven filters
+    user clicks button  Delete footnote
+    user clicks button  Confirm
+    user reloads page
+    user scrolls down  2000
+    user waits until button is enabled  Confirm data replacement
 
 Confirm data replacement
     [Tags]  HappyPath
-    user waits until page contains  Data blocks: OK
-    user waits until page contains  Footnotes: OK
     user clicks button  Confirm data replacement
     user waits until h2 is visible  Data replacement complete
+    # NOTE: I've noticed a bug where sometimes the Admin doesn't seem to take in the fact that the data replacement
+    # has finished. The below "user reloads page" keyword seems to prevent this behavior from happening
+    user reloads page
```

Comment:
I don't think this `user reloads page` is necessary. Line 400 works around the bug, so I think you can delete this and the comment on 408-9.

Generally, if there is a bug I'm working around, I'd just add a small comment linking to ticket - that's sufficient for someone in the future to know why its there so they can remove it if the bug is fixed.